docs/setup/permissions.md

```diff
@@ -15,36 +18,55 @@ with careful consideration for both battery life and security - only opening con
 
 #### External Storage Modification
 
-Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need 
+### Read your contacts
+
+In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
+need access to your contacts.
+
+### Modify your contacts
+
+In order to note that a contact has been emailed, we need write-access.
+
+## Storage
+
+### Read the contents of your USB storage
+### Modify or delete the contents of your USB storage
+
+Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need
 access to do so.
 
-### Extra/Custom
+## Photos / Media / Files
 
-#### Read Contacts
+### Read the contents of your USB storage
+### Modify or delete the contents of your USB storage
 
-In order to mark e-mail that comes from your contacts and use your contacts to provide email address auto-completion, we
-need access to your contacts
+Attachments and settings can be loaded and saved onto external storage. In order read and write these files we need
+access to do so.
 
-#### Write Contacts
+## Other
 
-In order to note that a contact has been emailed we need write-access.
+### Read sync settings
 
-#### Control Vibration
+In order to fetch contacts properly we need this permission.
```

Comment:
This permission is necessary to get the global auto-sync setting. See [ContentResolver#getMasterSyncAutomatically()](https://developer.android.com/reference/kotlin/android/content/ContentResolver#getMasterSyncAutomatically()).

Reply:
I wasn't quite sure what to write here as I'm not 100% on which content providers the app uses

Reply:
It has nothing to do with content providers. The global auto-sync setting (on my Android 11: *Settings app: Accounts > Automatically sync app data*) can be used as a flag to enable/disable background sync in K-9 Mail (see *Settings > General settings > Network > Background sync*).